recipes/gnuplot/meta.yaml

```diff
@@ -0,0 +1,49 @@
+{% set pkgname = "gnuplot" %}
+{% set version = "5.0.5" %}
+{% set sha256 = "25f3e0bf192e01115c580f278c3725d7a569eb848786e12b455a3fda70312053" %}
+package:
+  name: {{ pkgname }}
+  version: {{ version }}
+
+source:
+  fn: {{ pkgname }}-{{ version }}.tar.gz
+  url: https://sourceforge.net/projects/{{ pkgname }}/files/{{ pkgname }}/{{ version }}/{{ pkgname }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  skip: True  # [win]
+
+requirements:
+  build:
+    - pkgconfig
+    - ncurses 5.9*
+    - readline 6.2*
+    - libgd
+    - cairo 1.14.*
+    - qt 4.8.*
+    - pango >1.10
+  run:
+    - ncurses 5.9*
+    - readline 6.2*
+    - libgd
+    - cairo 1.14.*
+    - qt 4.8.*
+    - pango >1.10
```

Comment:
It is not there because we never really tested that. That is why I think it may be safer to pin than to start getting a mix of `pango` versions and having to debug it later :smile: